osu.Framework.Font/Graphics/Shaders/RepeatMovingBackgroundShader.cs

```diff
@@ -22,32 +24,42 @@ public class RepeatMovingBackgroundShader : InternalShader, IHasCurrentTime, IHa
 
     public float Mix { get; set; } = 1f;
 
-    public override void ApplyValue()
+    private IUniformBuffer<RepeatParameters>? repeatParametersBuffer;
+
+    public override void ApplyValue(IRenderer renderer)
     {
         if (Texture == null)
             return;
 
-        Texture.Bind(1);
-
-        var unitId = TextureUnit.Texture1 - TextureUnit.Texture0;
-        GetUniform<int>(@"g_RepeatSample").UpdateValue(ref unitId);
-
-        var textureCoord = Texture.GetTextureRect().TopLeft;
-        GetUniform<Vector2>(@"g_RepeatSampleCoord").UpdateValue(ref textureCoord);
+        // todo: think about how to upload the texture.
+        // Texture.Bind(1);
+        // var unitId = TextureUnit.Texture1 - TextureUnit.Texture0;
+        // GetUniform<int>(@"g_RepeatSample").UpdateValue(ref unitId);
```

Comment:
Oops, should deal with upload the texture to the shader also.

Reply:
Note that it's follow the code in the https://github.com/ppy/osu-framework/commit/9ec6711a909cf36ad7c82f12405a20071bf975a5#diff-ec8681ce3076c551c70eeebdc49697c9efb11b7d6f1e10bf0240235dbff7f7d3R122-R138
But it's removed 😢

Reply:
Will leave this broken part to the #323 because it might be hard to fix now.